Add b version of getting started guide for split testing (#3211)
pulumi/pulumi-hugo: themes/default/content/docs/get-started/b.md

---
title_tag: Get Started with Pulumi
meta_desc: Step-by-step guides for creating, deploying, and managing infrastructure with
           Pulumi on the cloud using your favorite language.
title: Start
h1: Get started with Pulumi
no_on_this_page: true
meta_image: /images/docs/meta-images/docs-meta.png
block_external_search_index: true
---

> Welcome! We are glad you chose Pulumi to help you automate your cloud infrastructure. Follow the steps below
> to get started with your first Pulumi project:

Reviewer: ```suggestion
Welcome! We're glad you chose Pulumi to help you automate your cloud infrastructure. Follow the steps below to get started with your first Pulumi project.
```

## Install the CLI

{{< install-pulumi >}}
{{% notes "info" %}}
All Windows examples in this tutorial assume you are running in PowerShell.
{{% /notes %}}
{{< /install-pulumi >}}

> ## Setup State Management

Author: 👍

> Pulumi stores metadata about your infrastructure so that it can manage your cloud resources. This metadata is called state. Each stack has its own state, and state is how Pulumi knows when and how to create, read, delete, or update cloud resources.

Author: 👍

Pulumi stores state in a backend of your choosing. A backend is an API and storage endpoint used by the CLI to coordinate updates, and read and write stack state whenever appropriate. Backend options include:

> - **Service**: a managed cloud experience using the online or self-hosted Pulumi Cloud application

Author: 👍

If you already have a self-managed backend set up you can login via the `login` command:

<div class="highlight">
    <pre class="chroma"><code class="language-bash" data-lang="bash">$ pulumi login $BACKEND_URL</code></pre>
</div>

If you do not have a self-managed backend set up, please refer to [Using a Self-Managed Backend](/docs/concepts/state/#using-a-self-managed-backend) documentation for information on how to set one up.

{{< /choosable >}}

## Install Language Runtime

Select your language:

{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" / >}}

{{% choosable language "javascript,typescript" %}}
{{< install-node >}}
{{% /choosable %}}

{{% choosable language python %}}
{{< install-python >}}
{{% /choosable %}}

{{% choosable language go %}}
{{< install-go >}}
{{% /choosable %}}

{{% choosable language "csharp,fsharp,visualbasic" %}}
{{< install-dotnet >}}
{{% /choosable %}}

{{% choosable language "java" %}}
{{< install-java >}}
{{% /choosable %}}

{{% choosable language "yaml" %}}
{{< install-yaml >}}
{{% /choosable %}}

> ## Pick a Starter Project
> 
> Pick from the starter projects below to start creating, modifying, and destroying infrastructure with Pulumi:

Reviewer: Nits, but "choose" sounds a teeny bit nicer than "pick", IMO, and I'd suggest removing "modifying and destroying", since the user won't "start" by doing either of those. 

```suggestion
## Choose a starter project

Choose from the list of starter projects below to start creating infrastructure with Pulumi:
```

Author: 👍
